tests/test_git_clone.rs

```diff
@@ -87,3 +87,91 @@ fn test_git_clone() {
     Error: Destination path exists and is not an empty directory
     "###);
 }
+
+#[test]
+fn test_git_clone_colocate() {
+    let test_env = TestEnvironment::default();
+    let git_repo_path = test_env.env_root().join("source");
+    let git_repo = git2::Repository::init(git_repo_path).unwrap();
+
+    // Clone an empty repo
+    let stdout = test_env.jj_cmd_success(
+        test_env.env_root(),
+        &["git", "clone", "source", "empty", "--colocate"],
+    );
+    insta::assert_snapshot!(stdout, @r###"
+    Fetching into new repo in "$TEST_ENV/empty"
+    Nothing changed.
+    "###);
+
+    // Clone a non-empty repo
```

Comment:
Should say "Set up a non-empty Git repo", maybe? It doesn't seem to clone a repo.